ui.json

```diff
@@ -12,6 +12,11 @@
       "matchUpdateTypes": ["patch", "minor"],
       "groupName": "MUI X",
       "updateNotScheduled": false
+    },
```

Comment:
Considering not all UIs are using Recoil, can this go within those UIs specifically vs making this global?

Reply:
Every UI other than identity is using recoil - my thinking was to skip across all then have an override in each repo that makes the change instead of needing to skip in every UI other than identity.

I'm fine doing the skip in each repo though

Reply:
Looks like *just* identity-ui is not using Recoil :(. That's really a shame that we went all in on this lib.

Reply:
Let's skip in the UIs vs global.